test/standalone/test-debuglet.js

```diff
@@ -17,9 +17,10 @@
 
 var assert = require('assert');
 var request = require('../auth-request.js');
-var logger = require('@google/cloud-diagnostics-common').logger;
-var config = require('../../src/config.js').debug;
+var loggerModule = require('@google/cloud-diagnostics-common').logger;
+var defaultConfig = require('../../src/config.js').debug;
 var Debuglet = require('../../src/agent/debuglet.js');
+var extend = require('extend');
```

Comment:
Since we already have lodash could we use `_.assignIn` instead?
https://lodash.com/docs/4.17.2#assignIn